### PR TITLE
Simplify strip for new packages

### PR DESCRIPTION
**Pull request by alesapin**
Changelog category (leave one):
- Not for changelog (changelog entry is not required)

### CODE REVIEW

File: cmake/strip_binary.cmake [hunk 1]
```diff
@@ -11,16 +11,43 @@ macro(clickhouse_strip_binary)
...
-     COMMAND bash ${ClickHouse_SOURCE_DIR}/cmake/strip.sh ${STRIP_BINARY_PATH} ${STRIP_DESTINATION_DIR} ${OBJCOPY_PATH} ${READELF_PATH}
-     COMMENT "Stripping clickhouse binary" VERBATIM
+       COMMAND mkdir -p "${STRIP_DESTINATION_DIR}/lib/debug/"
+       COMMAND mkdir -p "${STRIP_DESTINATION_DIR}/bin"
+       COMMAND cp "${STRIP_BINARY_PATH}" "${STRIP_DESTINATION_DIR}/bin/${STRIP_TARGET}"
+       COMMAND "${OBJCOPY_PATH}" --only-keep-debug --compress-debug-sections "${STRIP_DESTINATION_DIR}/bin/${STRIP_TARGET}" "${STRIP_DESTINATION_DIR}/lib/debug/${STRIP_TARGET}.debug"
+       COMMAND chmod 0644 "${STRIP_DESTINATION_DIR}/lib/debug/${STRIP_TARGET}.debug"
+       COMMAND "${STRIP_PATH}" --remove-section=.comment --remove-section=.note "${STRIP_DESTINATION_DIR}/bin/${STRIP_TARGET}"
+       COMMAND "${OBJCOPY_PATH}" --add-gnu-debuglink "${STRIP_DESTINATION_DIR}/lib/debug/${STRIP_TARGET}.debug" "${STRIP_DESTINATION_DIR}/bin/${STRIP_TARGET}"
+       COMMENT "Stripping clickhouse binary" VERBATIM
```
Comment (Felixoid) [inline cmake/strip_binary.cmake:26]: It looks like it's been executed many times. https://github.com/ClickHouse/ClickHouse/runs/5665394978?check_suite_focus=true#step:6:9957

Interesting, why

File: cmake/strip_binary.cmake [hunk 2]
```diff
@@ -11,16 +11,43 @@ macro(clickhouse_strip_binary)
...
    )
 
    install(PROGRAMS ${STRIP_DESTINATION_DIR}/bin/${STRIP_TARGET} DESTINATION ${CMAKE_INSTALL_BINDIR} COMPONENT clickhouse)
    install(DIRECTORY ${STRIP_DESTINATION_DIR}/lib/debug DESTINATION ${CMAKE_INSTALL_LIBDIR} COMPONENT clickhouse)
```
Comment (Felixoid) [inline cmake/strip_binary.cmake:30]: Ah, I get it. These commands install directory 3 times

**Comment by Felixoid** [comment]
@mergifyio rebase

**Comment by alesapin** [comment]
Ok, now everything much simpler.

**Pull request merged**